src/fhir/api.ts

```diff
@@ -0,0 +1,31 @@
+import {patientSchema} from "../components/reports/formDataValidation";
+import {createPatientEntry} from "./resources";
+
+/**
+ * Create a report bundle
+ * @param patient FHIR patient entity
+ */
+export const bundleRequest = (patient: typeof patientSchema) => {
+  return {
+    url: "/",
+    method: "POST",
+    headers: {'Content-Type': 'application/fhir+json;charset=UTF-8'},
+    body: JSON.stringify(createBundle(patient)),
+  };
+}
+
+export const createBundle = (patient: typeof patientSchema) => {
+  const mrn = patient?.identifier?.at(0)?.value;
+  const patientEntry = createPatientEntry(patient);
+
+  return {
+    resourceType: "Bundle",
+    type: "transaction",
+    entry: [{
+      resource: patientEntry,
+      resourceType: "Patient",
+      request: {method: "PUT", url: `Patient?identifier=${mrn}`}
+    }]
+  };
+}
```

Comment:
Again, would an error get caught in your `onSuccessfulSubmitHandler` function (e.g. if the transaction failed)? Should we show some sort of error message to the user in this case?

Reply:
we certainly should!

Reply:
I like the idea in the next review to have a component to do this - presumably we also should have this as an error boundary so going to resolve here and use discussion in #15